velox/exec/tests/AggregationTest.cpp

```diff
@@ -342,6 +345,68 @@ class AggregationTest : public OperatorTestBase {
     }
   }
 
+  // Inserts 'key' into 'order' with random bits and a serial
+  // umber. The serial number makes repeats of 'key' unique and the
+  // random bits randomize the order in the set.
+  void insertRandomOrder(
+      int64_t key,
+      int64_t serial,
+      folly::F14FastSet<uint64_t>& order) {
+    // The word has 24 bits of grouping key, 8 random bits and 32 bits of serial
+    // number.
+    order.insert(
+        ((folly::Random::rand32(rng_) & 0xff) << 24) | key | (serial << 32));
+  }
+
+  // Returns the key from a value inserted with insertRandomOrder().
+  int32_t randomOrderKey(uint64_t key) {
+    return key & ((1 << 24) - 1);
+  }
+
+  void addBatch(
+      int32_t count,
+      RowVectorPtr rows,
+      BufferPtr& dictionary,
+      std::vector<RowVectorPtr>& batches) {
+    std::vector<VectorPtr> children;
+    dictionary->setSize(count * sizeof(vector_size_t));
+    children.push_back(BaseVector::wrapInDictionary(
```

Comment:
nit: there is a helper method wrapInDictionary which doesn't require to pass nullptr for nulls